#goal definition
float64 depth_setpoint
---
#result definition
float64 depth_final
---
#feedback
float64 depth_error
